src/backend/TrafficCourts/Staff.Service/Services/JJDisputeService.cs

```diff
@@ -301,15 +331,42 @@ private async Task<string> GetStatuteDescriptionAsync(JJDisputedCount count)
             return string.Empty;
         }
     }
+
+    [LoggerMessage(EventId = 0, Level = LogLevel.Warning, EventName = "UserAssignedToTicketHasNoPartId", Message = "User assigned to ticket have PartId attribute in Keycloak")]
```

Comment:
I think this message should be "User assigned to ticket has no PartId attribute in Keycloak".